erclient/client.py

```diff
@@ -213,9 +213,9 @@ def time_converter(t):
             response = fn(self._er_url(path), data=body,
                           headers=headers, params=params)
 
-        if response and response.ok:
+        if response and response.ok and response.json() is not None:
```

Comment:
It's some well patched code below, so we do need to accept into this if block response and response.ok.

```
if response and response.ok:
    res_json = response.json()
    if res_json and 'data' in res_json and res_json['data']:
        return res_json['data']
    return res_json
```

Reply:
I see that. Makes sense. I have made the change.